lib/logstorage/block.go

```diff
@@ -41,42 +41,43 @@ func (b *block) reset() {
 
 // uncompressedSizeBytes returns the total size of the original log entries stored in b.
 //
-// It is supposed that every log entry has the following format:
-//
-// 2006-01-02T15:04:05.999999999Z07:00 field1=value1 ... fieldN=valueN
-func (b *block) uncompressedSizeBytes() uint64 {
-	rowsCount := uint64(b.Len())
-
-	// Take into account timestamps
-	n := rowsCount * uint64(len(time.RFC3339Nano))
+// It uses JSON format to calculate the size as if each log entry were represented as JSON.
+func (b *block) uncompressedSizeBytes() int {
+	rowsCount := b.Len()
```

Comment:
Please add a reference to EstimatedJSONRowLen() here in the comment, and also add a comment with the reference to the `block.uncompressedSizeBytes()` at the beginning of the `EstimatedJSONRowLen()` function. These cross-references are needed in order to make sure these functions are kept in sync in the future. Also please add a test, which calls both functions for the same set of logs and verifies they return similar results.